roles/argocd/templates/certificate.j2

```diff
@@ -0,0 +1,12 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: nginx-cert
+  namespace: argocd
+spec:
+  dnsNames:
+  - {{ domain }}
```

Comment:
Please standardize the indentation for list items in yaml.

Reply:
Fixed